doc/news/changes/minor/20190807Sanchez-Vizuet

```diff
@@ -0,0 +1,4 @@
+Add references to the introduction of step 51
```

Comment:
Take a look at the other files in this direction -- they usually start with something like "Changed:". The empty line you have below should contain the text `<br>`.